src/OnboardComputers/OnboardComputersManager.cpp

```diff
@@ -55,6 +58,34 @@ void OnboardComputersManager::_mavlinkMessageReceived(const mavlink_message_t& m
     }
 }
 
+void OnboardComputersManager::_checkTimeouts()
+{
+    if (_onboardComputers.isEmpty())
+        return;
+
+    auto iter=_onboardComputers.begin();
+    while (iter!=_onboardComputers.end())
+    {
+        auto &compIter=iter.value();
+        // Check all computers for timeout. If has some, emitting timeout on his ID, and removing it form list.
+        if (compIter.lastHeartbeat.elapsed()>_timeoutCheckInterval*2){
+            uint8_t compID=iter.key();
+            iter=_onboardComputers.erase(iter);
+            qCDebug(OnboardComputersManagerLog)<<"Deleting onboard computer: "<<compID <<", due to timeout.";
+            if (_currentComputerIndex==compID){
+                if (_onboardComputers.isEmpty()){
+                    _currentComputerIndex=0;
+                    emit currentComputerChanged(0);
+                }else
+                    setCurrentComputer(_onboardComputers.first().compID);
+            }
+            emit onboardComputerTimeout(MAV_COMP_ID_ONBOARD_COMPUTER + compID - 1);
+            continue;
```

Comment:
And here. Format to
```
            if (_currentComputerIndex == compID) {
                if (_onboardComputers.isEmpty()) {
                    _currentComputerIndex = 0;
                    emit currentComputerChanged(0);
                } else {
                    setCurrentComputer(_onboardComputers.first().compID);
                }
            }
            emit onboardComputerTimeout(MAV_COMP_ID_ONBOARD_COMPUTER + compID - 1);
            continue;
            ```
Also, do not forget to use bracers after `else`. It's much more easy to read and extend afterwards